src/main/java/com/google/firebase/messaging/TopicManagementResponse.java

```diff
@@ -122,5 +122,10 @@ public int getIndex() {
     public String getReason() {
       return reason;
     }
+
+    @Override
+    public String toString() {
+      return "Error{" + "index=" + index + ", reason='" + reason + '\'' + '}';
```

Comment:
Use `MoreObjects.toStringHelper()` here. See `FirebaseApp.toString()` for an example.